notebooks/100_datamodules/104_tiling.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p><s>how many pixels must be crossed in order to take the second tile</s> the spatial distance in pixels between adjacent tiles</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>